examples/wasi-http-rust-streaming-outgoing-body/src/lib.rs

```diff
@@ -1,34 +1,32 @@
-use {
-    self::wasi::http::types::{
-        Fields, IncomingRequest, Method, OutgoingBody, OutgoingRequest, OutgoingResponse,
-        ResponseOutparam, Scheme,
-    },
-    anyhow::{bail, Result},
-    futures::{stream, FutureExt, SinkExt, StreamExt, TryStreamExt},
-    sha2::{Digest, Sha256},
-    spin_sdk::wasi_http_component,
-    std::str,
-    url::Url,
+use anyhow::{bail, Result};
+use futures::{stream, SinkExt, StreamExt, TryStreamExt};
+use spin_sdk::wasi_http::send;
+use spin_sdk::wasi_http::{
+    Fields, IncomingRequest, IncomingResponse, Method, OutgoingBody, OutgoingRequest,
+    OutgoingResponse, ResponseOutparam, Scheme,
```

Comment:
I know this already existed but would it be possible to change the name of `ResponseOutparam` before freezing this?  (Or is this low-level WIT detail and the SDK will provide a more conventionally named wrapper?)

Reply:
This comes directly from `wasi-http`. We could alias to another name, but I want to be careful about creating a potentially confusing rename.